test/library/packages/LinearAlgebra/performance/dot-perf.chpl

```diff
@@ -20,35 +19,36 @@ config type eltType = real;
 const nbytes = numBytes(eltType);
 
 proc main() {
-  const Space = {1..n};
-  const BlockSpace = Space dmapped Block(boundingBox=Space);
+  const dom = {1..n, 1..n};
   
-  var BA : [BlockSpace] real = [i in BlockSpace] (i / (1000000));
+  var BA : [dom] real = [(i,j) in dom] (j / (1000000));
 
   var t: Timer;
 
   if !correctness {
     writeln('============================');
     writeln('Dense Dot Performance Test');
     writeln('============================');
-    writeln('iters : ', iters);
-    writeln('n     : ', n);
-    writeln('numLocales     : ', numLocales);
-    writeln('MB    : ', (nbytes*n) / 10**6);
+    writeln('iters                  : ', iters);
+    writeln('square matrix size     : ', n);
+    writeln('MB                     : ', (nbytes*n*n) / 10**6);
     writeln();
   }
-  
-  var d: eltType;
 
+  var result = dot(BA, BA);
```

Comment:
How about we time this dot and do 1 less iteration below, making the correctness/performance paths more overlapped? E.g.,

```chpl
t.start();
var result = dot(BA, BA);
t.stop();

for 1..<iters {
  t.start();  
  result = dot(BA, BA);
  t.stop();
}
```

Reply:
Sounds good to me